src/App.js

```diff
@@ -22,12 +37,13 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  onChange={this.takeInput}
```

Comment:
Если это инпут, то это всегда
```suggestion
                  onChange={this.handleChange}
```